dietpi/dietpi-software

```diff
@@ -13981,6 +13981,14 @@ _EOF_
 
 		# - IPversion preference: https://github.com/Fourdee/DietPi/issues/472
 		/DietPi/dietpi/func/dietpi-set_hardware preferipversion "$(grep -m1 '^[[:blank:]]*CONFIG_PREFER_IPVERSION=' /DietPi/dietpi.txt | sed 's/^.*=//')"
+		
```

Comment:
APT cache does not exist during ```FirstRun_Automation_Set``` on first run if:
- No DietPi updates applied

I know we set ```-1``` to image versions, but I'd probably go with putting this in here for end of first run, just to make sure:
https://github.com/Fourdee/DietPi/blob/02a751ac0ea7e9e640486973aa26098abdcc3fca/dietpi/dietpi-software#L13805

Reply:
@MichaIng 

I'll edit PR and make the changes, will also remove all previous patches for microcode.